implementations/rust/ockam/ockam_command/src/project/get_credential.rs

```diff
@@ -0,0 +1,45 @@
+use clap::Args;
+
+use ockam::credential::Credential;
+use ockam::Context;
+use ockam_api::clean_multiaddr;
+use ockam_core::api::Request;
+use ockam_multiaddr::MultiAddr;
+
+use crate::node::NodeOpts;
+use crate::util::{node_rpc, Rpc};
+use crate::{stop_node, CommandGlobalOpts, Result};
+
+#[derive(Clone, Debug, Args)]
+pub struct GetCredentialCommand {
+    #[clap(flatten)]
+    node_opts: NodeOpts,
+
+    #[clap(long, short)]
+    to: MultiAddr,
+}
+
+impl GetCredentialCommand {
+    pub fn run(self, opts: CommandGlobalOpts) {
+        node_rpc(rpc, (opts, self));
+    }
+}
+
+async fn rpc(
+    mut ctx: Context,
+    (opts, cmd): (CommandGlobalOpts, GetCredentialCommand),
+) -> Result<()> {
+    async fn go(
+        ctx: &mut Context,
+        opts: &CommandGlobalOpts,
+        cmd: &GetCredentialCommand,
+    ) -> Result<()> {
+        let to = clean_multiaddr(&cmd.to, &opts.config.get_lookup()).unwrap();
+        let mut rpc = Rpc::new(ctx, opts, &cmd.node_opts.api_node)?;
+        rpc.request_to(Request::post("/credential"), &to).await?;
```

Comment:
Does it work without secure channel?

Reply:
Katharina is working on the `project` resolution helper for multiaddr. Once we have that, when the multiaddr includes a project (`/project/default/service/forwarder_to_blue/service/api`), it will retrieve it and create the secure channel. So, we should probably wait until we have that in place, or merge it as it is and add that later.

Reply:
If the PR is ready, I suggest merging it with secure channels and add secure channels as part of the project change @spacekookie has been working on. 

This way we don't have to spend time rebasing and making the PR ready for merge again